openfeature/event_executor_test.go

```diff
@@ -975,7 +988,7 @@ func TestEventHandler_HandlersRunImmediately(t *testing.T) {
 			rsp <- e
 		}
 
-		AddHandler(ProviderReady, &callback)
+		//AddHandler(ProviderReady, &callback) // TODO: hard to test, because callback is executed on handler registration
```

Comment:
Is this TODO still valid?

Reply:
unfortunately, yes. The RDY callback comes from `emitOnRegistration`. Now that i cannot mock the state via the provider implementation the testing this becomes a bit harder.